app/models/photo.js

```diff
@@ -0,0 +1,14 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  photoFileName: DS.attr('string'),
+  photoContentType: DS.attr('string'),
+  photoFileSize: DS.attr('integer'),
+  isPrivate: DS.attr('boolean'),
+  photoUpdatedAt: DS.attr('datetime'),
+  soda: DS.belongsTo('soda')
+  // user: DS.belongsTo('user')
+
+  // How do I declare this in ember model? Couldn't find an answer. has_attached_file :photo
+
+});
```

Comment:
can we pull this out into another branch? you could cut one for https://trello.com/c/SGmfb06q/41-spike-new-soda-form-photo-upload where photo uploading is added, but sodas should not require a photo. If soda is complaining we need a photo, we should fix that, since photos are optional

if you cut a branch for 41, please add a comment to the trello